### DISCUSSION TITLE
Wizard Steps forward and backward

### DISCUSSION THREAD

**Discussion by jcgmalave** [filamentphp/filament#4055] [Help] [answered]
it is not possible that in the steps of the wizard you can advance forwards and backwards, indistinctly

**Comment by danharrin** [comment, marked as answer]
Wdym "indistinctly"?

**Reply by jcgmalave** [reply]
right now I can only, when I'm in the edit section, go back when clicking on the steps, to those that I have already advanced, but to advance I can only do it by pressing the NEXT button, I don't know if I'm explaining myself well; I would like to know if it is possible to go forward or backward, by clicking on the steps and also by pressing the NEXT and PREVIOUS buttons

**Reply by danharrin** [reply]
`skippable()` on the Wizard (in the latest version)

**Reply by jcgmalave** [reply]
Thanks a LOT!!!! GENIUS
